project_proposal/tex/mainpage.tex

```diff
@@ -155,10 +166,13 @@ \subsection{Use Case Diagram}
 \end{figure}
 
 \subsection{Supplementary Requirements}
+FURPS is a model for classifying functional and non-functional requirements. The
+acronym stand for:
```

Comment:
Is the acronym missing?

Reply:
Not quite sure what I was thinking here. It's been rewritten, does it make sense now?

Reply:
It does make senese